### DISCUSSION TITLE
Hide Tooltip when the mouse leaves the anchor, even if hovering over the Tooltip itself

### DISCUSSION THREAD

**Discussion by hmaesta** [ariakit/ariakit#4316] [Questions]
Currently, the Tooltip remains open if the user leaves the anchor, but hovers over the Tooltip itself:

<img src="https://github.com/user-attachments/assets/634c9817-c6ec-43e5-8fb4-619a9c80f02f" width="150"/>

ㅤ
Is there a way to disable this behavior? The Tooltip should close whenever the mouse leaves the anchor.

I tried disabling focus, pointer events and selection, but with no success:

```tsx
<Ariakit.Tooltip
  autoFocusOnHide={false}
  autoFocusOnShow={false}
  disabled={true}
  focusable={false}
  disablePointerEventsOnApproach={false}
  portal={true}
  portalElement={document.getElementById("portal")!}
  style={{ pointerEvents: "none", userSelect: "none" }}
... 
```

[Sandbox](https://codesandbox.io/p/sandbox/9zjj5h)

**Comment by hmaesta** [comment]
Got it by making it a controlled component — but still open to suggestions!

```tsx
const [isOpen, setIsOpen] = useState(false);

const tooltip = Ariakit.useTooltipStore({
  open: isOpen,
  setOpen: setIsOpen,
})

<Ariakit.TooltipAnchor 
  (...)
  onMouseOver={() => setIsOpen(true)}
  onMouseOut={() => setIsOpen(false)}
>
```

**Reply by benrodrs** [reply]
It's possible, but it's designed that way for a reason. You might end up with an inaccessible element.

Some magnification tools use the mouse position to zoom in on elements. By hiding the tooltip, some people may not be able to read it. Additionally, users won't be able to select the text inside the tooltip to copy or translate, for example.
